add-tourpackage.php

```diff
@@ -191,67 +182,67 @@
                     <div class="col-12 mt-3">
                         <div class="row">
                             <div class="col-12 col-md-4 col-lg-4">
-                                <label>Total Milage</label>
+                                <label>Total Milage (KM)</label>
                             </div>
                             <div class="col-12 col-md-8 col-lg-8">
-                                <input type="text" class="w-100">
+                                <input type="number" class="w-100" id="milage">
                             </div>
                         </div>
                     </div>
                     <div class="col-12 mt-2">
                         <div class="row">
-                            <div class="col-6">
+                            <div class="col-12 col-md-4 col-lg-4">
                                 <span>Main Image:</span>
                             </div>
-                            <div class="col-6">
+                            <div class="col-12 col-md-8 col-lg-8 mt-2">
                                 <div class="row">
                                     <div class="col-12">
-                                        <input class="form-control w-100" type="file" id="formFile" accept=".png, .jpg, .jpeg, .heic" onclick="">
+                                        <input class="form-control w-100" type="file" id="main-image" accept=".png, .jpg, .jpeg, .heic" onclick="mainImagePreview();">
```

Comment:
Remove the `.heic` file extension. We only accept `.png` , `.jpg` , `.jpeg` file extensions for images.

Reply:
done